can't use TOMCAT_8_5_24 for testing

Excerpt from the jenkins' console log. 
> downloadfile:
      [get] Getting: http://downloads.sourceforge.net/checkstyle/checkstyle/6.17/checkstyle-6.17-all.jar
      [get] To: /root/tomcat-build-libs/download-1562367266.tmp
      [get] http://downloads.sourceforge.net/checkstyle/checkstyle/6.17/checkstyle-6.17-all.jar permanently moved to http://downloads.sourceforge.net/project/checkstyle/OldFiles/6.17/checkstyle-6.17-all.jar
      [get] http://downloads.sourceforge.net/project/checkstyle/OldFiles/6.17/checkstyle-6.17-all.jar moved to https://master.dl.sourceforge.net/project/checkstyle/OldFiles/6.17/checkstyle-6.17-all.jar
BUILD FAILED
/tomcat85/build.xml:2604: The following error occurred while executing this line:
/tomcat85/build.xml:2806: Redirection detected from http to https. Protocol switch unsafe, not allowed.

It's because `sourceforege.net` automatically forwards the http request to https.  Ant building logic prohibits this behavior.  tomcat repo has updated to https since TOMCAT_8_5_30.  Is that okay I send a PR to bump the version? 

https://github.com/AdoptOpenJDK/openjdk-tests/blob/f3856860ea0a30c936240d4959284443bb636181/thirdparty_containers/tomcat/dockerfile/tomcat-test.sh#L52

--- a/thirdparty_containers/tomcat/dockerfile/tomcat-test.sh
+++ b/thirdparty_containers/tomcat/dockerfile/tomcat-test.sh
@@ -49,7 +49,7 @@ cd /tomcat85
 ls .
 pwd
 echo "Building tomcat" && \
-git checkout TOMCAT_8_5_24 && \
+git checkout TOMCAT_8_5_30 && \
 cp build.properties.default build.properties && \
 ant && \
 

